addons/l10n_be/i18n/de.po

```diff
@@ -264,36 +205,22 @@ msgstr "Kontenplanvorlage"
 
 #. module: l10n_be
 #. odoo-python
-#: code:addons/l10n_be/models/template_be_ass.py:0
+#: code:addons/l10n_be/models/template_be_asso.py:0
 #, python-format
 msgid "Associations and Foundations"
-msgstr "Vereinigungen und Stiftungen"
+msgstr ""
```

Comment:
Why is the translation deleted?

Reply:
PR was still WIP, I had to run some scripts to avoid this kind of loss